compete.html

```diff
@@ -130,12 +130,12 @@ <h3>Building a robot</h3>
       <a name="signup"></a><h2>Want to compete?</h2>
       <div class="column m-8-12 m-offset-2-12 text-left">
         <p>
-          Please register your interest below, we'll let you know when formal entries open later this year.
+          Please register your interest below. We'll let you know once your place is confirmed.
```

Comment:
This could be read that it's a rubber-stamp process and everyone gets a place. Maybe phrase like:
```suggestion
          Please register your interest below. We'll let you know places are confirmed.
```

Reply:
Are you missing a word in this suggestion? "We'll let you know _once_ places are confirmed"?

Reply:
:nothing-to-see-here:
```suggestion
          Please register your interest below. We'll let you know once places are confirmed.
```

Reply:
https://github.com/srobo/website/pull/156/commits/6ce910ae2d75d77f2cf822048d03392dd6f398ad